components/VM_Arm/src/modules/vuart_init.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <vmlinux.h>
+#include <camkes.h>
+
+#include <sel4vmmplatsupport/plat/devices.h>
+
+extern void *serial_getchar_buf;
+
+//this matches the size of the buffer in serial server
+#define BUFSIZE 4088
```

Comment:
Can we use some shares header structure here, so this is kept in sync? Is there a chance to rename this `serial_shared_page` instead of `serial_getchar_buf`. Otherwise a comment here would be nice that explains where the 4088 byte come from.

Reply:
This is identical to the `virtio_console` module. Can we track an issue to create a shared header for a serial server interface?

Reply:
Thanks, Axel!